src/lib/dialog/dialog.scss

```diff
@@ -0,0 +1,52 @@
+@import '../core/style/elevation';
+@import '../a11y/_a11y';
+
+
+$md-dialog-padding: 24px !default;
+$md-dialog-border-radius: 2px !default;
+$md-dialog-max-width: 80vw !default;
+$md-dialog-max-height: 65vh !default;
```

Comment:
The `max-width` is the same that Material 1 used, the `max-height` is arbitrary. M1 has 80% for the `max-height` as well, although that's for the entire dialog, whereas here we only have it on the `md-dialog-content`.